src/test/app/Vault_test.cpp

```diff
@@ -2766,18 +2766,6 @@ class Vault_test : public beast::unit_test::suite
                 "malformedRequest");
         }
 
-        {
```

Comment:
Why was this test case removed?

Reply:
It was failing and seemed wrong. I don't think a sequence of 0 is a malformed request, I think the ledger object just won't exist.

Reply:
Unfortunately, I don't have knowledge if this is ok or not, let's have someone from `rippled` to confirm it's ok to remove this test.